### PR TITLE
Add support for c.resource(...).createOrReplaceAnd().waitUntilReady()

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Readiable.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package io.fabric8.kubernetes.client.dsl;
+
+public interface Readiable {
```
Comment (jimmidyson) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Readiable.java]: Typo: `Readable`

**Comment by jimmidyson** [comment]
Oh and license check too

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Readiable.java (hunk 1 above)
Comment (iocanel) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Readiable.java]: We are after Ready-able not readable.

Comment (jimmidyson) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Readiable.java]: Sorry I meant `Readyable` but as it's not a word it doesn't matter I guess.

**Comment by jimmidyson** [comment]
[merge]

**Pull request merged**